node/engine/store/durablestore.go

```diff
@@ -26,6 +27,7 @@ type DurableStore struct {
 	consensusChannels  *buntdb.DB
 	channelToObjective *buntdb.DB
 	vouchers           *buntdb.DB
+	blocks             *buntdb.DB
```

Comment:
My preference would be something like `lastBlockSeen` instead of `blocks`, since that's all we're really storing in `blocks`. Not a huge deal either way.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/1581/commits/1185ea1f6ba21612424ee2678b86ee7a4bff284e